zebrad/tests/acceptance.rs

```diff
@@ -23,7 +23,7 @@ use color_eyre::eyre::Result;
 use eyre::WrapErr;
 use tempdir::TempDir;
 
-use std::{convert::TryInto, env, fs, io::Write, path::Path, path::PathBuf, time::Duration};
+use std::{collections::HashSet, convert::TryInto, env, path::Path, path::PathBuf, time::Duration};
```

Comment:
Why'd these get moved?

Reply:
So people will use the `with_config` and `replace_config` utility functions, rather than doing filesystem operations directly.

Outside of those utility functions, there's only a single `fs::create_dir` in the tests themselves.

Reply:
sounds good